src/components/LicenseForm/LicenseForm.js

```diff
@@ -133,7 +134,7 @@ class LicenseForm extends React.Component {
         <Button
           id={id}
           type="submit"
-          disabled={this.props.pristine || this.props.submitting}
+          disabled={this.props.pristine || this.props.submitting || this.props.invalid}
```

Comment:
Can you destructure these props at this point? Starting to get all `this.props...`-y